cloud-template/src/main/java/com/sequenceiq/cloudbreak/cloud/template/compute/CloudFailureHandler.java

```diff
@@ -145,7 +145,7 @@ private void doRollbackAndDecreaseNodeCount(AuthenticatedContext auth, List<Clou
                             futures.clear();
                         }
                     } catch (Exception e) {
-                        LOGGER.debug("Resource can not be deleted. Reason: {} ", e.getMessage());
+                        LOGGER.debug("Resource can not be deleted. Reason: {} ", e.getMessage(), e);
```

Comment:
Logger:
public void debug(String msg, Throwable t);

you need  to format the string because `e` would be swallowed (but would be printed only as a toString() if it had a placeholder)

Reply:
Thanks @bergerdenes !